skore/tests/integration/cli/test_create.py

```diff
@@ -2,26 +2,26 @@
 
 import pytest
 from skore.cli.create_project import (
-    ImproperProjectName,
-    ProjectAlreadyExists,
+    InvalidProjectNameError,
+    ProjectAlreadyExistsError,
     ProjectCreationError,
-    ProjectNameTooLong,
+    ProjectNameTooLongError,
     __create,
     validate_project_name,
 )
 
 test_cases = [
     (
         "a" * 250,
-        (False, ProjectNameTooLong()),
+        (False, ProjectNameTooLongError()),
```

Comment:
Based on my previous comment, here it would throw an `InvalidProjectNameError`.

Reply:
done